exporter/splunkhecexporter/README.md

```diff
@@ -56,7 +56,7 @@ exporters:
     timeout: 10s
     # Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS. Defaults to false.
     insecure_skip_verify: false
-    # Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS. Defaults to false.
+    # Whether to disable client transport security for the exporter's connection. Defaults to false.
```

Comment:
if `insecure` is true then `TLSClientSetting.LoadTLSConfig()` returns `nil` for `tls.Config`. The result is set in `http.Transport.TLSClientConfig`. See [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/ea2ea16b48f01f7402474c1f811134fa70e4bff4/exporter/splunkhecexporter/exporter.go#L95).

```go
	// TLSClientConfig specifies the TLS configuration to use with
	// tls.Client.
	// If nil, the default configuration is used.
	// If non-nil, HTTP/2 support may not be enabled by default.
	TLSClientConfig *tls.Config
```

If it's `nil` then the default TLS config is used which really has nothing to do with insecure.

To disable TLS for HTTP clients like splunkhec you have to change https to http in the URL. So best I can tell this setting should only be used for gRPC.

Reply:
I see. So the best way to avoid confusion is just to remove the line from the README?

Reply:
Or maybe explicitly say not to use it.

Reply:
ok, I removed mentions of that option to reduce noise